package.json

```diff
@@ -36,7 +36,7 @@
     "node": ">=4.2.4"
   },
   "dependencies": {
-    "mongoose": "^5.1.6"
+    "mongoose": "^5.2.5"
```

Comment:
Adjust this to lock exactly to 5.2.5 instead of with the range operator.

Also, please adjust the package name to no longer be just `mongoose-paginate` - change it to `@remhealth/mongoose-paginate` - I think this was done in the build that's on the nexus now, but was never checked in....

Set the repo URL to `https://github.com/remarkable-health/mongoose-paginate.git` instead of the upstream repo

Finally, set the version number in package.json to 0.0.0 instead of 5.1.1. The build server will set the version before publishing to nexus.